nix/scripts.nix

```diff
@@ -0,0 +1,76 @@
+{ evalService
```

Comment:
I'm not so sure yet about these scripts.
Our users would like to have convenience scripts for connecting to mainnet/testtnet, but possibly in a way where nix is optional.
Can we just do the NixOS service in this PR, and review these convenience scripts in a new PR?

Reply:
I think those scripts can/are also to be used as entry-point for bitte clusters containers, (cc @blaggacao for confirmation) so would be needed as part of the flake.

Reply:
Yes, these scripts are required to run the wallet in nomad clusters, such as ERC20 converter.

Reply:
OK, thanks @jbgi, @disassembler.